### PR TITLE
Update prepublish gate to allow beta publishing from branches

### PR DESCRIPTION
**Pull request by sophistifunk**
Update to prepublish.sh that allows beta versions to be published without matching master

### CODE REVIEW

**Comment by sophistifunk** [comment]
From a fork because I don't have write access to this repo. Also can't set PR tags :)

File: prepublish.sh [hunk 1]
```diff
@@ -6,13 +6,18 @@ set -e
 
 REPO=js-builder
 
-# Make sure that the local branch tip revision is the same 
-# as upstream master.
-LOCAL_TIP=$(git rev-parse HEAD)
-REMOTE_TIP=$(git ls-remote https://github.com/jenkinsci/$REPO master | awk '{ print $1 }')
-
-if [ $LOCAL_TIP != $REMOTE_TIP ]
+if [ "$npm_config_tag" != "latest" ]
```
Comment (tfennelly) [inline prepublish.sh]: `$npm_config_tag` needs to be in the `env`? If so, maybe hint at that in the message at line 20?

Comment (sophistifunk) [inline prepublish.sh]: It's added by npm, so the user doesn't need to do anything.

Comment (tfennelly) [inline prepublish.sh]: Ah cool .... I'll merge

Comment (tfennelly) [inline prepublish.sh]: I was wondering how this might be possible actually ... I never went to the trouble of finding out though ... thanks 😄

**Pull request merged**